ee/debug/checkups/services_windows.go

```diff
@@ -228,10 +232,38 @@ func gatherServices(z *zip.Writer, serviceManager *mgr.Mgr) error {
 	return nil
 }
 
+// gatherServiceManagerEvents uses Get-WinEvent to fetch the service manager logs. This might be newer than Get-EventLog
+func gatherServiceManagerEvents(ctx context.Context, z *zip.Writer) error {
+	out, err := z.Create("eventlog-Get-WinEvent.json")
+	if err != nil {
+		return fmt.Errorf("creating eventlog-Get-WinEvent.json: %w", err)
+	}
+
+	cmdArgs := []string{
+		"Get-WinEvent",
+		`-FilterHashtable @{LogName='System'; ProviderName='Service Control Manager'}`,
+		"|",
+		"ForEach-Object { $_.Message }",
+	}
```

Comment:
The other option was filtering Service Control Manager events on ones that specifically had to do with the Kolide service. I think we might want that here too -- it would keep us from having to sift through irrelevant events, and from collecting data we don't need. What do you think? (Is it possible to filter like that with `Get-WinEvent`?)

Reply:
I do, yes! Happy to run some tests in powershell if you'd like

Reply:
If you can, that'd be great. I don't have a handy windows host :|

Reply:
Ok, I took a look!

An unrelated item I noticed is that this won't include date/timestamp, which I figure we may want for correlation with launcher logs later. We could do the same pipe to `Format-Table -Wrap -AutoSize` that the command below does instead of filtering on the `Message` here. That gives us date, timestamp, event code, error level, and message -- not an overwhelming number of columns.

As far as I can tell from [Get-WinEvent docs](https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.diagnostics/get-winevent?view=powershell-7.4), we can't filter on Message the same way we do for `Get-EventLog`. If we want a similar implementation, then this works for me:

```
Get-WinEvent  -FilterHashtable @{LogName='System'; ProviderName='Service Control Manager'} | 
    Where {$_.Message.Contains('LauncherKolideK2Svc')} | 
    Format-Table -Wrap -AutoSize
```

Let me know if you want me to try out anything else.

Reply:
I found how to search in the message body, and tossed it into json. This seems to work on a windows test machine I have. Does it work on yours?

Reply:
The filtering works -- I needed to put single quotes around the service name, though. `@{LogName='System'; ProviderName='Service Control Manager'; Data='LauncherKolideK2Svc'}`

Reply:
(The `ConvertTo-Json` is nice!)

Reply:
Okay. I think I just added single ticks

Reply:
One caveat about `ConvertTo-Json` -- the timestamps are in a windows native format, it's ms since the epoch. Just something to know when translating them